src/Fixer/Import/NoUnusedImportsFixer.php

```diff
@@ -145,18 +145,20 @@ private function isImportUsed(Tokens $tokens, NamespaceAnalysis $namespace, arra
 
                 if (
                     0 === strcasecmp($shortName, $token->getContent())
-                    && !$prevMeaningfulToken->isGivenKind([T_NS_SEPARATOR, T_CONST, T_OBJECT_OPERATOR])
+                    && !$prevMeaningfulToken->isGivenKind([T_NS_SEPARATOR, T_CONST, T_OBJECT_OPERATOR, T_DOUBLE_COLON])
                 ) {
                     return true;
                 }
 
                 continue;
             }
 
-            if ($token->isComment() && Preg::match(
-                '/(?<![[:alnum:]])(?<!\\\\)'.$shortName.'(?![[:alnum:]])/i',
-                $token->getContent()
-            )) {
+            if ($token->isComment()
+                && Preg::match(
+                    '/(?<![[:alnum:]])(?<!\\\\)'.$shortName.'(?![[:alnum:]])/i',
```

Comment:
Remove `/i`.

Reply:
the regex remain the same as it was. php imports are case insensitive.

Reply:
> php imports are case insensitive.

Not necessarily, actually. When it comes to autoloading, case matters if one has a case sensitive filesystem. So, while class fully qualified names are case insensitive, if the autoloader is later fired, case matters.